### PR TITLE
statistic：新增statistic组件，用于数值展示和倒计时

### PR DESCRIPTION
**Pull request by webvs2**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/statistic/src/index.vue [hunk 1]
```diff
@@ -0,0 +1,208 @@
+<!-- eslint-disable @typescript-eslint/no-this-alias -->
```
Comment (chenxch) [inline packages/components/statistic/src/index.vue:1]: Need to cancel the eslint rule.

File: packages/components/statistic/src/index.vue [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        <span class="suffix">
+          <slot name="suffix">
+            {{ suffix }}
+          </slot>
+        </span>
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
```
Comment (chenxch) [inline packages/components/statistic/src/index.vue:32]: Please use setup syntax.

File: packages/components/statistic/src/index.vue [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+import { defineComponent } from 'vue'
+import _ from 'lodash'
```
Comment (chenxch) [inline packages/components/statistic/src/index.vue:34]: Please don't fully import `lodash`.

File: packages/components/statistic/src/index.vue [hunk 4]
```diff
@@ -0,0 +1,208 @@
+<!-- eslint-disable @typescript-eslint/no-this-alias -->
+<template>
+  <div>
+    fsa
```
Comment (chenxch) [inline packages/components/statistic/src/index.vue:4]: Please delete unnecessary content.

**Review by chenxch — COMMENTED** [review]
If you really want to submit a component, please refer to the implementation and specifications of other components, rather than simply copying your own code.

File: packages/components/statistic/src/index.vue (hunk 1 above)
Comment (webvs2) [reply packages/components/statistic/src/index.vue:1]: Ok, I will adjust it as soon as possible

File: packages/components/statistic/src/index.vue (hunk 2 above)
Comment (chenxch) [reply packages/components/statistic/src/index.vue:32]: yes.

Comment (webvs2) [reply packages/components/statistic/src/index.vue:32]: Thank you. I will adjust it as soon as possible

**Comment by sxzz** [comment]
暂不接受新组件，请先阅读 https://github.com/element-plus/element-plus/issues/7324

**Comment by webvs2** [comment]
> 暂不接受新组件，请先阅读#7324



> 暂不接受新组件，请先阅读#7324

需要讨论吗

**Comment by sxzz** [comment]
@webvs2 是的，建议加入discord讨论，并查看 https://github.com/element-plus/element-plus/issues/7324 的例子。

**Comment by webvs2** [comment]
@sxzz  讨论网址无法打开，是需要？

**Pull request closed without merging**